### ISSUE TITLE
Possible ambiguity in padding documentation for some methods

### ISSUE THREAD

**Issue by ozancaglayan** [pytorch/pytorch#3092]
In `torch.nn.Conv*`, the `padding` parameter is documented as **Zero-padding added to both sides of the input** but in `torch.nn.functional.Conv*` this sentence changes to **implicit zero padding on the input. Can be a single number or a tuple** [0].

At first I thought that this was a difference between the nature of the modules but `torch.nn.functional.avg_pool1d` also mentions about **both sides** while `torch.nn.avg_pool2d/3d` revert back to [0]. So is there really a difference about how the padding is applied between `torch.nn` and `torch.nn.functional` or is it the documentation which is not synchronized between modules/methods?

**Comment by SsnL** [comment]
`torch.nn.Conv*` padding can indeed be tuples. They call `torch.nn.functional.conv*` functions underneath. This is a documentation inconsistency. Thanks for lettings us know!

For example, see below:
```
>>> conv_module = nn.Conv2d(3, 3, padding = (1, 3))
>>> x = Variable(torch.zeros(1, 3, 3, 3))
>>> conv_module(x).shape
torch.Size([1, 3, 3, 7])
>>> conv_module = nn.Conv2d(3, 3, (3, 3), padding = 3)
>>> conv_module(x).shape
torch.Size([1, 3, 7, 7])
>>> conv_module = nn.Conv2d(3, 3, (3, 3), padding = (3, 3))
>>> conv_module(x).shape
torch.Size([1, 3, 7, 7])
>>> torch.nn.functional.conv2d(x, Variable(torch.zeros(3, 3, 3, 3)), padding = (1, 3)).shape
torch.Size([1, 3, 3, 7])
>>> torch.nn.functional.conv2d(x, Variable(torch.zeros(3, 3, 3, 3)), padding = 3).shape
torch.Size([1, 3, 7, 7])
>>> torch.nn.functional.conv2d(x, Variable(torch.zeros(3, 3, 3, 3)), padding = (3, 3)).shape
torch.Size([1, 3, 7, 7])
```

**Comment by SsnL** [comment]
Oh I though you were talking about the single number or tuple part. What do you mean by [0]? I don't see any [0] on the doc.

**Comment by ozancaglayan** [comment]
Sorry for the mess. I meant about double sidedness of padding. Is it always the case?

**Comment by SsnL** [comment]
Oh yes, the paddings are always symmetric.

**Comment by ozancaglayan** [comment]
Okay. So the explanation that should appear for every padding argument of Conv* et *Pool* methods be it in torch.nn or torch.nn.functional should be: 
  **Zero-padding added to both sides of the input. Can be a single number or a tuple.**

If you approve, I'd be happy to send a PR.

**Comment by SsnL** [comment]
Thanks for willing to contribute! Unfortunately, I’m already writing a PR
to update the doc. Sorry about this.

**Comment by soumith** [comment]
closed via #3110
